src/pages/Login.tsx

```diff
@@ -0,0 +1,142 @@
+import { useState } from 'react'
+import { signInWithEmailAndPassword} from 'firebase/auth'
+import { auth } from '../firebase'
```

Comment:
utilzar @ para los imports
```suggestion
import { auth } from '@/firebase'
```